README.md

```diff
@@ -19,6 +19,9 @@ An introduction to the NVIDIA Container Runtime is also covered in our [blog pos
 **If you have a custom `/etc/docker/daemon.json`, the `nvidia-docker2` package might override it.**  
 
 #### Ubuntu 14.04/16.04/18.04, Debian Jessie/Stretch
+
+Ubuntu Linux will install `docker.io` by default which isn't the latest version of docker. This implies that you will need to pin the version of NVIDIA docker. [See more information here](https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#how-do-i-install-20-if-im-not-using-the-latest-docker-version).
```

Comment:
Hello sorry one last correction and I will merge this!

`Ubuntu will install `docker.io` by default which isn't the latest version of Docker Engine`
`This implies that you will need to pin the version of nvidia-docker`

Thanks!

Reply:
I had followed your correction and this PR had been updated.